datasetinsights/io/gcs.py

```diff
@@ -210,3 +210,31 @@ def _upload_folder(
             self._upload_file(
                 local_path=full_path, bucket=bucket, key=object_key
             )
+
+    def get_most_recent_blob(self, url=None, bucket_name=None, key=None):
+        """Get the last updated blob in a given bucket under given prefix
+                bucket_name (str): gcs bucket name
```

Comment:
Please follow google style docstring to make sure this is parsed correctly. 
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
done